README.md

```diff
@@ -1,89 +1,96 @@
 
-Supported versions: 2.3, 2.4, 2.5.
+-  [Ruby](https://www.ruby-lang.org/en/documentation/installation)—Supported versions: 2.3, 2.4, 2.5
+-  [RubyGems](https://rubygems.org/pages/download)—packaging system for Ruby
+-  [rbenv](https://github.com/rbenv/rbenv)—Optional, but useful for managing one or more Ruby versions concurrently
 
-macOS and Ubuntu usually provide Ruby by default.
-
-To check Ruby version on your environment, run in your terminal:
+#### To check your Ruby version:
 
 ```shell
 $ ruby -v
 ruby 2.5.3p105 (2018-10-18 revision 65156) [x86_64-darwin18]
 ```
 
-See the [Ruby website](https://www.ruby-lang.org/en/documentation/installation) for installation instructions.
+## Local build using Jekyll
 
-*Windows installation using package manager*:  See [Build DevDocs in Windows](#build-devdocs-in-windows)
+You must clone the repository and install the dependencies before building the site. You only need to perform the following steps once:
 
-#### Packaging system
+1.  Clone the repository.
 
-RubyGems is a packaging system in Ruby.
+    ```shell
+    $ git clone git@github.com:magento/devdocs.git
+    ```
 
-Update the RubyGems:
+1.  Change to the `devdocs/` directory.
 
-```shell
-gem update --system
-```
+1.  Install the project dependencies using Bundler.
+
+    ```shell
+    $ bundle install
+    ```
 
-### Installation
+    > **Note**  
+    >[Bundler](https://bundler.io/), installed by RubyGems, is a tool that manages installation dependencies.
 
-1. Clone the repository.
-2. Install the project dependencies using Bundler:
+#### To build the site using a Jekyll command:
+
+The following Jekyll command builds the HTML and opens the site in a browser.
 
 ```shell
-$ bundle install
-<truncated output>
-Using parallel 1.12.1
-Using yell 2.0.7
-Using html-proofer 3.9.3
-Using progressbar 1.10.0
-Using verbal_expressions 0.1.5
-Using jekyll-algolia 1.4.8
-Using launchy 2.4.3
-Bundle complete! 5 Gemfile dependencies, 99 gems now installed.
-Use `bundle info [gemname]` to see where a bundled gem is installed.
+$ bundle exec jekyll serve --open-url
 ```
 
->**Note**
->[Bundler](https://bundler.io/) is a tool to manage installation dependencies.
->It is installed as a part of the [packaging system update](#packaging-system).
+```
+Configuration file: /Users/user/github/devdocs/_config.yml
+            Source: /Users/user/github/devdocs
+       Destination: /Users/user/github/devdocs/_site
+ Incremental build: enabled
+      Generating... 
+                    done in 43.806 seconds.
+ Auto-regeneration: enabled for '/Users/user/github/devdocs'
+    Server address: http://127.0.0.1:4000/
+  Server running... press ctrl-c to stop.
+```
 
-Once you have completed preparing your environment, you can build locally and review the site in your browser.
+The site server URL http://127.0.0.1:4000/ opens in a browser so you can preview the content.
 
-### Build locally
+Press **Ctrl**+**C** in the serve terminal to stop the server.
```

Comment:
Press **Ctrl**+**C** in the server terminal to stop the server.

Reply:
I wasn't sure if you meant server terminal -- I've not seen "serve terminal".

Reply:
It is a Jekyll "serve", but that's okay. I'll remove it.

Reply:
To explain that further, one can have multiple terminals open, even when hosting an incremental build. Specifying which terminal by referring to it as the one with the serve.